### PR TITLE
Comment input

### PR DESCRIPTION
**Pull request by JOHNFLEURIMOND**

### CODE REVIEW

File: services-js/commissions-app/src/client/common/CommentInput.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
```
Comment (fionawhim) [inline services-js/commissions-app/src/client/common/CommentInput.tsx:1]: Good use of a new component!

File: services-js/commissions-app/src/client/common/CommentInput.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  value: string;
+  onChange: any;
+  onBlur: any;
+}
+
+export default class Comments extends React.Component<Props> {
+  render() {
+    return (
+      <div className="txt">
+        <label className="txt-l" />
```
Comment (fionawhim) [inline services-js/commissions-app/src/client/common/CommentInput.tsx]: Do you want `{this.props.title}` in here?

Comment (fionawhim) [reply services-js/commissions-app/src/client/common/CommentInput.tsx]: Oh, I can see you're not really using it for this element. In that case, I'd recommend either:

 - Make `title` an optional prop and don't show `<label>` at all when it’s not there.
 - Remove `title` and `<label>` altogether from this component.

As it is the `title` prop is just ignored by this component, which is bound to confuse someone down the road.

**Review by fionawhim — CHANGES_REQUESTED** [review]
Looking pretty good. Just one thing to fix with the `title` prop.

File: services-js/commissions-app/src/client/common/CommentInput.tsx (hunk 2 above)
Comment (JOHNFLEURIMOND) [reply services-js/commissions-app/src/client/common/CommentInput.tsx]: I made it an option, Well I made it optional the best way I know how

File: services-js/commissions-app/src/client/common/CommentInput.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  onBlur: any;
+}
+
+export default class Comments extends React.Component<Props> {
+  render() {
+    return (
+      <div className="txt">
+        <textarea
+          rows={10}
+          title={this.props.title ? this.props.title : undefined}
```
Comment (fionawhim) [inline services-js/commissions-app/src/client/common/CommentInput.tsx]: Where did the `<label>` go? Take a look at `TextInput` to see how it used `this.props.title` to put in the label field.

File: services-js/commissions-app/src/client/common/CommentInput.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+          className={
+            this.props.error ? 'txt-f txt-f--sm txt-f--err' : 'txt-f txt-f--sm'
+          }
+          placeholder={this.props.placeholder}
+          value={this.props.value}
+          id="FeedbackForm-${this.props.name}"
+          onChange={this.props.onChange}
+          onBlur={this.props.onBlur}
+        />
+      </div>
```
Comment (fionawhim) [inline services-js/commissions-app/src/client/common/CommentInput.tsx:35]: I just noticed you're not using `this.props.error` in this `render()` method.

File: services-js/commissions-app/src/client/common/CommentInput.tsx (hunk 3 above)
Comment (JOHNFLEURIMOND) [reply services-js/commissions-app/src/client/common/CommentInput.tsx]: My apologies, i miss read what you said.

File: services-js/commissions-app/src/client/common/CommentInput.tsx (hunk 4 above)
Comment (JOHNFLEURIMOND) [reply services-js/commissions-app/src/client/common/CommentInput.tsx:35]: I made it optional, I took it off of the apply.tsx unless you want it off altogether

Comment (fionawhim) [reply services-js/commissions-app/src/client/common/CommentInput.tsx:35]: I'd expect you'd want the same error rendering as `TextInput` has.

File: services-js/commissions-app/src/client/common/CommentInput.tsx (hunk 3 above)
Comment (fionawhim) [reply services-js/commissions-app/src/client/common/CommentInput.tsx]: No worries. Can you get it back in?

File: services-js/commissions-app/src/client/common/CommentInput.tsx (hunk 4 above)
Comment (JOHNFLEURIMOND) [reply services-js/commissions-app/src/client/common/CommentInput.tsx:35]: Kayla said she didn't want this required but if you want that error I can put it in

**Review by fionawhim — APPROVED** [review]
LGTM!

**Pull request merged**